snow/networking/handler/message_queue.go

```diff
@@ -32,6 +32,10 @@ type Message struct {
 	EngineType p2p.EngineType
 }
 
+func (m Message) String() string {
+	return m.Message().String()
+}
+
```

Comment:
Because `message.InboundMessage` specifies a `String() string` function, I don't think we need to implement this here (it will be inherited).